pkg/sfu/downtrack.go

```diff
@@ -294,7 +294,7 @@ func NewDownTrack(params DowntrackParams) (*DownTrack, error) {
 		kind:               kind,
 		codec:              codecs[0].RTPCodecCapability,
 		pacer:              params.Pacer,
-		maxLayerNotifierCh: make(chan struct{}, 20),
+		maxLayerNotifierCh: make(chan struct{}, 1),
```

Comment:
This is just a notifier. So, don't need a deep channel. I had a different implementation before (I was queuing actual quality changes, but later made it a notification channel only).